dashactivity.php

```diff
@@ -193,7 +193,7 @@ public function hookDashboardData($params)
 					WHERE (g.id_customer IS NULL OR g.id_customer = 0)
 						'.Shop::addSqlRestriction(false, 'c').'
 						AND cp.`time_end` IS NULL
-					AND TIME_TO_SEC(TIMEDIFF(\''.pSQL(date('Y-m-d H:i:00', time())).'\', cp.`time_start`)) < 900
+						AND cp.`time_start` > now() - interval 900 second
```

Comment:
The problem is `NOW()` cannot be cached in MySQL :thinking: 
```suggestion
						AND cp.`time_start` > \''.pSQL(date('Y-m-d H:i:00', time())).'\' - interval 900 second
```

Reply:
Hi,

the query change every second, because of usage of time(). So there is no need of MySQL Query Cache here.
Furthermore, the MySQL Query Cache is a deprecaced feature :
    https://dev.mysql.com/doc/refman/5.7/en/query-cache-configuration.html

Reply:
It changes every minutes in MySQL because of `H:i:00`
Even if it's deprecated, it's really useful for old MySQL versions because they are not optimised.
And because we support old versions of MySQL, we need to keep this kind of behavior. 

Maybe one day @PrestaShop/prestashop-core-developers will accept to remove the support of old MySQL database :sweat_smile:

Reply:
@PierreRambaud Not actually the case ;) So we can't accept this PR with `NOW()`.

Reply:
Note : MariaDB is not removing query cache, and it's not a old DB :)
Aurora on AWS actually fixed the mutex contention issue related to the query cache. 
So having the Suggested change instead of NOW() is the good approach and should be fixed ASAP :)

Reply:
oh BTW it's already fixed by https://github.com/PrestaShop/dashactivity/commit/d743f089809c581d5f4a8c6e25dfb7689f6c3a52 :)